packages/pn-personafisica-webapp/public/locales/it/recapiti.json

```diff
@@ -84,19 +84,19 @@
     "phone-new-code": "Non l’hai ricevuto? Controlla che il numero di cellulare sia corretto o",
     "new-code-link": "generane uno nuovo",
     "email-added": "Indirizzo e-mail",
-    "phone-added": "Numero di cellulare",
+    "sms-added": "Numero di cellulare",
     "valid-email": "Indirizzo e-mail non valido",
     "valid-phone": "Numero di cellulare non valido",
     "email-added-successfully": "Indirizzo e-mail aggiunto correttamente",
     "phone-added-successfully": "Numero di cellulare aggiunto correttamente",
```

Comment:
is phone-added-successfully or sms-added-successfully?

Reply:
is ok `phone-added-successfully`

Reply:
can we use sms...?